src/encoded/static/components/app.js

```diff
@@ -1471,13 +1476,25 @@ class ContentRenderer extends React.PureComponent {
  */
 class BodyElement extends React.PureComponent {
 
+    static getDerivedStateFromProps(props, state){
+        var stateChange = { 'lastHref' : props.href };
+        if (state.isFullscreen && stateChange.lastHref !== state.lastHref){
```

Comment:
"// Unset full screen if moving away to different pathname."
This comment is still relevant, I would keep it.